### ISSUE TITLE
Sharing option (Image) of rankings in technology always exports the Satisfaction image and not the others

### ISSUE THREAD

**Issue by ze-gomes** [StateOfJS/StateOfJS-2020#27]
Pretty self explanatory, was trying to export a few images of usage of technologies, adn selecting the Image option always exports the satisfaction image. Not the others.

**Comment by SachaG** [comment]
Yes, that's a limitation of our current system. It'll probably be tough to fix, sorry :(
